packages/react-native-client/src/hooks/usePeers.ts

```diff
@@ -68,40 +63,38 @@ export type Peer<
   /**
    * a list of peers's video and audio tracks
    */
-  tracks: Track<VideoTrackMetadataType | AudioTrackMetadataType>[];
+  tracks: Track[];
 };
 
+function addIsActiveToTracks<MetadataType extends Metadata>(
+  peers: ReadonlyArray<Peer<MetadataType>>,
+): Peer<MetadataType>[] {
+  return peers.map((peer) => ({
+    ...peer,
+    tracks: peer.tracks.map((track) => ({
+      ...track,
+      isActive:
+        (track as { metadata?: TrackMetadata })?.metadata?.active ?? true,
```

Comment:
does the web sdk also pass `active` and `type` in metadata?

Reply:
Web SDK often rely on just removing track, so it is not always needed. But @kamil-stasiak is working on changes to also remove metadata handling from part of web SDK. So at some point, we could sync mobile and web approach